src/core/api/index.js

```diff
@@ -0,0 +1,20 @@
+import { Schema, arrayOf, normalize } from 'normalizr';
+
+// FIXME: Load this from something.
+const API_HOST = 'https://addons-dev.allizom.org';
```

Comment:
I tried getting this to work but it didn't work on the first try and I'm not sure if we'll just need `process.env.NODE_ENV` or more than that so I'm going to leave it for the other issue.